ios/GoodDollar/AppDelegate.m

```diff
@@ -48,7 +48,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
   self.window = [self initializeWindow:rootViewController];
 
-  [self initializeBackgroundFetch];
+  // Define UNUserNotificationCenter
+  UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
```

Comment:
move this and startMonitorNotifications back to the [self initializeNotifications]

Reply:
will do